src/components/animated/_colored-on-hover.less

```diff
@@ -0,0 +1,67 @@
+@colors:
```

Comment:
What's this variable for?

Reply:
It has color hex code declaration in an array named colors

Reply:
Is it the colors of the buttons (color for blue-btn, yellow-btn, etc...)? If so please use the variables for those colors. You can find them in `_variables.less`

Reply:
It's a long task means remapping the variables of _variables.less into local variables of the colors array. Can't this be okay...

Reply:
instead of using the blue button as a hex using it as a variable like `@blue`. I'm not sure why it's a long task?